Sources/Controllers/MessagesViewController.swift

```diff
@@ -208,6 +211,11 @@ UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
             let cell = messagesCollectionView.dequeueReusableCell(LocationMessageCell.self, for: indexPath)
             cell.configure(with: message, at: indexPath, and: messagesCollectionView)
             return cell
+        case .audio:
+            let cell = messagesCollectionView.dequeueReusableCell(AudioMessageCell.self, for: indexPath)
+            cell.configure(with: message, at: indexPath, and: messagesCollectionView)
+            audioController.configureAudioCell(cell, message: message) // this is needed especily when the cell is reconfigure while is playing sound
```

Comment:
@moldovaniosif Stopping by again because I gave this more thought tonight. The only thing that concerns me here is `BasicAudioController` from a maintenance perspective. The reason this feature has had a lower priority to me is because I will never use it personally.

1. Is there any way to add Audio cells to MessageKit without the burden of the player?
2. If there's no way around this, can you be the code owner for this feature of MessageKit?

Reply:
1. I think it should be possible to move the `AudioController` outside the framework.
2. Yes, I can take the responsibility for this feature.

When I will find some time I will try to pull out the `AudioController` as @nathantannar4 mention letting inside the framework only the audio cell UI. Then I can make a new pull request for the `AudioController`. If you have some thoughts about how the `AudioController` should behave or how should be structured I am opened to any suggestions.